src/ape/managers/chain.py

```diff
@@ -720,7 +720,7 @@ def get(
 
             if proxy_info:
                 self._local_proxies[address_key] = proxy_info
-                return self.get(proxy_info.target)
+                return self.get(proxy_info.target, default)
```

Comment:
We pass optional arguments as kwargs:

```suggestion
                return self.get(proxy_info.target, default=default)
```